portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/PropertiesList.testcase

```diff
@@ -15,6 +15,15 @@ definition {
 	tearDown {
 		var testPortalInstance = PropsUtil.get("test.portal.instance");
 
+		ACUtils.tearDownDXP();
+
+		if ("${testPortalInstance}" == "true") {
+			PortalInstances.tearDownCP();
+		}
+		else {
+			JSONGroup.deleteGroupByName(groupName = "Site Name");
+		}
+
```

Comment:
Hey @joshitagaki, I think not all tests will need this tearDown. I think you could just put your test to disconnect and delete the site.  What do you think?

Reply:
@ViniLopes20 I think the teardown is needed in case of the test case failing before reaching the end where the disconnect is supposed to happen. Otherwise we might have the connection between AC and DXP during the next test and an additional site.

Reply:
Yes @joshitagaki, I think you mentioned a good point. Thank you for that